abandonauth/routers/ui.py

```diff
@@ -135,7 +135,7 @@ async def list_developer_applications(request: Request) -> Response:
         "developer_apps.html",
         {
             "request": request,
-            "dev_apps": [x["id"] for x in dev_apps],
+            "dev_apps": [(x["name"], x["id"]) for x in dev_apps],
```

Comment:
I wonder if we could use a dictionary structure here, would make the HTML template usage clearer too.

```suggestion
            "dev_apps": [{"name": x["name"], "id": x["id"])} for x in dev_apps],
```

Or even just
```suggestion
            "dev_apps": [x for x in dev_apps],
```
although I prefer the first as we explicitly say what is available to the template.

Reply:
Thanks for review!